django_tables2/columns/templatecolumn.py

```diff
@@ -14,7 +15,9 @@ class TemplateColumn(Column):
     Arguments:
         template_code (str): template code to render
         template_name (str): name of the template to render
-        extra_context (dict): optional extra template context
+        context_object_name (str): name of the context variable to pas the record in, defaults to "record".
+        extra_context (dict): optional extra template context. If a callable is passed, it will be called with
+            optional record, table, value, bound_column arguments.
```

Comment:
```suggestion
        extra_context (dict): optional extra template context. Any callables passed will be called with the following 
            optional arguments: record, table, value, and bound_column.
```

Reply:
kan je meer dan 1 callable meegeven in de `dict`?

Reply:
gegeven mijn verwarring over extra_context hieronder kunnen we niet zeggen:
extra_context: accepts  dict or callable. If a called is passed it will be called with the following optional arguments: record, ...